test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeArrayTests.java

```diff
@@ -668,4 +668,49 @@ static public void testValueTypeAastore() throws Throwable {
       writeArrayElementWithSomeVTClassImplIf(holder);
       writeArrayElementWithSomeIdentityClassImplIf(holder);
    }
+
+	public static primitive class EmptyPrim {
+	}
+
+	public static value class EmptyVal {
+	}
+
+	static void copyBetweenEmptyPrimArrays(EmptyPrim[] arr1, EmptyPrim[] arr2) {
+		for (int i = 0; i < arr1.length; i++) {
+			arr1[i] = arr2[i];
+		}
+	}
+
+	static void compareEmptyPrimArrays(EmptyPrim[] arr1, EmptyPrim[] arr2) {
+		for (int i = 0; i < arr1.length; i++) {
+			assertEquals(arr1[i],arr2[i]);
```

Comment:
missing a space between two arguments for assertEquals: `assertEquals(arr1[i],arr2[i]);` => `assertEquals(arr1[i], arr2[i]);`

Reply:
Fixed in commit https://github.com/eclipse-openj9/openj9/commit/6fb2ec179c5c66a84d891308ec3016b83e982680